04-Multi-Band-Raster.Rmd

```diff
@@ -70,22 +65,36 @@ Read more about the `raster` package in R.</a>
 
 #About Raster Bands
 
-As discussed in the [Intro to Raster Data Lesson 00]( {{ base.url }} }}/R/Introduction-to-Raster-Data-In-R), a raster can contain 1 or more bands. To 
-work with multi-band rasters, we need to adjust how we import and plot our data. 
+As discussed in the [Intro to Raster Data Lesson 00]( {{ base.url }} }}NEON-R-Spatial-Raster/R/Introduction-to-Raster-Data-In-R), a raster can
+contain 1 or more bands. To work with multi-band rasters, we need to change how 
+we import and plot our data in several ways. 
 
-* To import multi band raster data we will use the `stack` function.
-* If our multi-band data are imagery that we wish to composit, we can use `plotRGB` 
-to plot a 3 band image raster.
+* To import multi band raster data we will use the `stack()` function.
+* If our multi-band data are imagery that we wish to composite (spatial data
+term essentially meaning, to view multiple bands together), we can use
+`plotRGB()` to plot a 3 band image raster.
+
+>Bonus: Bands? Layers?  Why are both names used? The term bands comes from the 
+>fact that the data contained within each *band* comes from a different region or
+>band of the electromagnetic spectrum.  When we are using multiple bands to 
+>create images we layer on band on top of another, hence, the name layers.  In a 
+>GIS context layers can things other than bands.  In Lesson 00 we added a lines 
+>shapefile and a polygon shapefile in one image.  One could say that the line 
+>layer was added to the polygon layer in the plot.  
 
 #About Multi Band Imagery
-A raster dataset can store multiple bands. One multi-band raster dataset that is  familiar to many of us is an image. A basic color image consists of three bands:
-red, green and blue. The pixel brightness for each band, when composited creates
-the colors that we see in an image.
+A raster dataset can store multiple bands. One multi-band raster dataset that is
+familiar to many of us is an image. A basic color image consists of three bands:
+red, green, and blue. The pixel brightness for each band, when composited creates
+the colors that we see in an image. 
 
-####we can plot each band of an 3 band image individually
+We can plot each band of a multi-band image individually. 
 
-```{r demonstrate-RGB-Image, echo=FALSE}
+>Note: In a typical GIS application, a single band would render as a single image
+>band using a grayscale color palette. We will thus use a grayscale palette to 
+>render individual bands.
```

Comment:
I think we should use the notice tag for these paragraphs - not the `>`
I think we could come up with `{ : .notice1}`  and `.notice2` to have 2 styles. the code is just MUCH simpler to manage.